README.rst

```diff
@@ -23,6 +23,25 @@ a `tax-deductible donation`_ to help the project pay for developer time, profess
 
 This project is bound by a `Code of Conduct`_.
 
+If you use MDAnalysis_ in your project consider lettting your users, and the world, know about it by displaying the MDAnalysis_ badge! |powered_by_MDA|
+
+.. raw:: html
```

Comment:
can't use raw, rewrite with standard reST.

Reply:
Do we have an all-round compatible way of displaying 'spoiler'-type blocks using standard reST? O I just leave this as a simple code block and hope whatever renders `README.rst` adds a bottom scroller?

Reply:
At least in GitHub's rendering the code-block approach seems to work. Should I create one block per language (Markdown, reST, html)? Will that be too crowded for README.rst as a landing page? I can have a single block with only the URL, but then there's no way to prod users into actually linking the badge to https://www.mdanalysis.org...

Maybe the best solution is to have an external resource (a gist? the blog post?) listing the implementation in different markups and we just link to it from the README?

Reply:
Now linked to the [website](https://www.mdanalysis.org/pages/citations/#powered-by-mdanalysis).